src/main/java/uk/gov/ons/census/fwmt/jobservice/consumer/queue/RmConfig.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.ons.census.fwmt.jobservice.consumer.queue;
```

Comment:
Im not sure where its configured but I just found out  we need to read off a queue called 'RM.Field'